Tests/SwiftDocCTests/Utility/SymbolExtensionsTests.swift

```diff
@@ -0,0 +1,73 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2023 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import XCTest
+import SymbolKit
+@testable import SwiftDocC
+
+class SymbolExtensionsTests: XCTestCase {
+
+    func testOffsetAdjustedForInterfaceLanguage() throws {
+
+        func lineListFrom(docs: String?) -> SymbolGraph.LineList? {
+            guard let str = docs else {
+                return nil
+            }
+            let range = SymbolGraph.LineList.SourceRange(
+                start: .init(line: 1, character: 2),
+                end: .init(line: 3, character: 4)
+            )
+            let line = SymbolGraph.LineList.Line(text: str, range: range)
+            return SymbolGraph.LineList([line])
+        }
+
+        func createSymbol(interfaceLanguage: String, docs: String? = nil) -> SymbolGraph.Symbol {
+            return SymbolGraph.Symbol(
+                identifier: .init(precise: "abcd", interfaceLanguage: interfaceLanguage),
+                names: .init(title: "abcd-in-\(interfaceLanguage)", navigator: nil, subHeading: nil, prose: nil),
+                pathComponents: ["documentation", "mykit", "abcd"],
```

Comment:
minor: Neither "documentation" not the module name are included in the path components that are emitted by Clang or the Swift compiler.

Reply:
makes sense; thx - removed those